russstation/code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -60,3 +60,19 @@
 	attack_force = 20
 	pick_speed = 0.2
 	sharp_result = TRUE
+
+/datum/reagent/confettium
+	name = "Confettium"
+	description = "A sick concoction of chemistry and clownery."
+	reagent_state = LIQUID
+	color = "#e03add" //rgb 224, 58, 221
+	taste_description = "paper cuts"
+
+/datum/reagent/confettium/reaction_turf(turf/T, reac_volume)
+	if(reac_volume >= 1)
+		if(!isspaceturf(T))
+			var/obj/effect/decal/cleanable/confetti/CONF = locate() in T.contents
+			if(!CONF)
+				CONF = new/obj/effect/decal/cleanable/confetti(T)
+			if(!QDELETED(CONF))
+				CONF.reagents.add_reagent(type, reac_volume)
```

Comment:
Could `CONF` get changed to capital `C` or `conf`? Using multiple all capital letters makes it look like a macro instead of a variable.

Reply:
Changed to `conf`